.readthedocs.yaml

```diff
@@ -0,0 +1,25 @@
+version: "2"
+
+build:
+  os: "ubuntu-22.04"
+  tools:
+    python: "3.10"
+  jobs:
+    post_install:
+      # Install latest botorch if not on a released version
+      - |
+        tag=$(eval "git name-rev --name-only --tags HEAD")
+        if [ $tag = "undefined" ]; then
+          pip install git+https://github.com/cornellius-gp/linear_operator.git
+          pip install git+https://github.com/cornellius-gp/gpytorch.git
+        fi
+
+python:
+  install:
+    - method: pip
+      path: .
+      extra_requirements:
+        - dev
```

Comment:
Does this need the allow lastest gpytorch env var?

Reply:
I don't think so. The order of operations here is different from how we usually install deps because ReadTheDocs doesn't have a way for us to conditionally set an environment variable. There's a `pre_install` step available to us but environment variables set there don't actually persist to the installation step.

My solution here was to install pinned deps first then conditionally install the latest versions as a `post_install` step.